src/entities/RfqmJobEntity.ts

```diff
@@ -1,46 +1,94 @@
 import { BigNumber } from '@0x/asset-swapper';
+import { RfqOrder } from '@0x/protocol-utils';
 import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
 
-import { RfqmJobStatus, StoredFee, StoredOrder } from '../utils/rfqm_db_utils';
-
 import { BigNumberTransformer } from './transformers';
 
+export interface StoredFee {
+    token: string;
+    amount: string;
+    type: 'fixed' | 'bps';
+}
+
+export enum RfqmJobStatus {
+    InQueue = 'inQueue',
+    Processing = 'processing',
+    Submitted = 'submitted',
+    Successful = 'successful',
+    Failed = 'failed',
+}
+
+export enum RfqmOrderTypes {
+    V4Rfq = 'v4Rfq',
+}
+
+export interface V4RfqStoredOrder {
+    type: RfqmOrderTypes.V4Rfq;
+    order: V4StringRfqOrderFields;
+}
+
+export interface V4RfqStoredOrder {
+    type: RfqmOrderTypes.V4Rfq;
+    order: V4StringRfqOrderFields;
+}
+
+export type StoredOrder = V4RfqStoredOrder;
+export type RfqmOrder = RfqOrder;
```

Comment:
nit, I would move this type back to db_utils since it's related to parsing the stored order into something more useful by the application

Reply:
dun